airflow/providers/amazon/aws/operators/glue.py

```diff
@@ -239,3 +243,253 @@ def on_kill(self):
+        running Airflow in a distributed manner and aws_conn_id is None or
+        empty, then default boto3 configuration would be used (and must be
+        maintained on each worker node).
+    :param region_name: AWS region_name. If not specified then the default boto3 behaviour is used.
+    :param verify: Whether or not to verify SSL certificates. See:
+        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html
+    :param botocore_config: Configuration dictionary (key-values) for botocore client. See:
+        https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html
+    """
+
+    aws_hook_class = GlueDataQualityHook
+    template_fields: Sequence[str] = ("name", "ruleset", "data_quality_ruleset_kwargs")
+
+    template_fields_renderers = {
+        "data_quality_ruleset_kwargs": "json",
+    }
+    ui_color = "#ededed"
+
+    def __init__(
+        self,
+        *,
+        name: str,
+        ruleset: str,
+        description: str = "AWS Glue Data Quality Rule Set With Airflow",
+        update_rule_set: bool = False,
+        data_quality_ruleset_kwargs: dict | None = None,
+        aws_conn_id: str | None = "aws_default",
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.name = name
+        self.ruleset = ruleset.strip()
+        self.description = description
+        self.update_rule_set = update_rule_set
+        self.data_quality_ruleset_kwargs = data_quality_ruleset_kwargs or {}
+        self.aws_conn_id = aws_conn_id
+
+    def validate_inputs(self) -> None:
+        if not self.ruleset.startswith("Rules") or not self.ruleset.endswith("]"):
+            raise AttributeError("RuleSet must starts with Rules = [ and ends with ]")
+
+        if self.data_quality_ruleset_kwargs.get("TargetTable"):
+            target_table = self.data_quality_ruleset_kwargs["TargetTable"]
+
+            if not target_table.get("TableName") or not target_table.get("DatabaseName"):
+                raise AttributeError("Target table must have DatabaseName and TableName")
+
+    def execute(self, context: Context):
+        self.validate_inputs()
+
+        config = {
+            "Name": self.name,
+            "Ruleset": self.ruleset,
+            "Description": self.description,
+            **self.data_quality_ruleset_kwargs,
+        }
+
+        if self.update_rule_set:
+            self.hook.update_glue_data_quality_ruleset(config)
+            self.log.info("AWS Glue data quality ruleset updated successfully")
+        else:
+            self.hook.create_glue_data_quality_ruleset(config)
+            self.log.info("AWS Glue data quality ruleset created successfully")
+
+
+class GlueDataQualityRuleSetEvaluationRunOperator(AwsBaseOperator[GlueDataQualityHook]):
+    """
+    Once you have a ruleset definition (either recommended or your own), you call this operation to evaluate the ruleset against a data source (Glue table).
```

Comment:
It feels weird to describe an operator with a sentence starting by "Once you have a". Suggestion: "Evaluate a ruleset against a data source (Glue table)".

Reply:
Agree, updated the changes.